### PR TITLE
Disable tracking in `/donate-widget` 

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* `<Seo/>` to have `scripts` prop, defaulted to our tracker scripts
* `/donate-widget` sets `Seo.scripts` to `[ ]` 

## Instructions on making this work
* open cognito window
* visit `localhost:4200/donate-widget/{x}` - see that popup `doesn't` appear
* for comparison, visit `staging/donate-widget/{x}` - see that popup `appears`
- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -62,13 +62,9 @@ export default function App() {
           />
           <Route
             path={donateWidgetRoutes.stripe_payment_status}
-            element={<StripePaymentStatus />}
+            element={<StripePaymentStatus isInWidget />}
           />
         </Route>
-        <Route
```
Comment (ap-justin) [inline src/App/App.tsx:68]: redundant route

File: src/pages/DonateThanks.tsx [hunk 2]
```diff
@@ -57,7 +60,9 @@ export default function DonateThanks({ widgetVersion = false }) {
 
       <p className="text-center text-navy-l1 mt-8 mb-2 text-[15px]">
         {state?.microdepositArrivalDate
-          ? `The microdeposit is expected to arrive at your nominated bank account on ${new Date(state.microdepositArrivalDate * 1000)}. You can access the bank verification link from the "Pending" tab on your`
+          ? `The microdeposit is expected to arrive at your nominated bank account on ${new Date(
```
Comment (ap-justin) [inline src/pages/DonateThanks.tsx:63]: not sure why this is formatted

File: src/pages/StripePaymentStatus.tsx [hunk 3]
```diff
@@ -23,8 +24,6 @@ export default function StripePaymentStatus() {
 
   const handleProcessing = useCallback(() => refetch(), [refetch]);
 
-  const isInWidget = window.location.pathname.includes(appRoutes.donate_widget);
```
Comment (ap-justin) [inline src/pages/StripePaymentStatus.tsx:26]: this is now set in container

**Review by SovereignAndrey — APPROVED** [review]
:clap: Love this idea of leveraging the Seo component.

**Pull request merged**